cnxarchive/views/content.py

```diff
@@ -153,6 +153,26 @@ def is_latest(id, version):
     return get_latest_version(id) == version
 
 
+def get_state(cursor, id, version):
+    """Determine the state of the content."""
+    args = '@'.join((id, version))
+
+    sql_statement = """
+    SELECT ms.statename
+    FROM modules as m
+    JOIN modulestates as ms
+    ON m.stateid=ms.stateid
+    WHERE ident_hash(uuid, major_version, minor_version) = %s
+    """
+
+    cursor.execute(sql_statement, vars=(args,))
+    res = cursor.fetchone()
+    if res is None:
+        return u'No Stateid in Modules table'
```

Comment:
Let's expose this as `None`. For the most part nothing on the front-end is going to be able to decipher the meaning of this since the terminology is very different on the front-end. Also, `None` or `null` will be more appropriate for javascript logic switching since the other 'state names` will be constant.